t/id-keyword-draft6.t

```diff
@@ -0,0 +1,45 @@
+use Mojo::Base -strict;
+use Test::Mojo;
+use Test::More;
+use JSON::Validator;
+
+my ($base_url, $jv, $t, @e);
+
+use Mojolicious::Lite;
+get '/person'           => 'person';
+get '/invalid-relative' => 'invalid-relative';
+
+$t = Test::Mojo->new;
+$jv = JSON::Validator->new(ua => $t->ua);
+
+eval {
+  $t->get_ok('/person.json')->status_is(200);
+  $base_url = $t->tx->req->url->to_abs->path('/');
```

Comment:
There's a little bit too much things inside this eval. The two lines above should never fail.

Reply:
I tried to keep it as close to the existing t/id-keyword-draft7.t as possible :p